integration/e2emimir/services.go

```diff
@@ -189,8 +189,9 @@ func NewCompactor(name string, consulAddress string, flags map[string]string, op
 			"-compactor.ring.store":           "consul",
 			"-compactor.ring.consul.hostname": consulAddress,
 			// Startup quickly.
-			"-compactor.ring.wait-stability-min-duration": "0",
-			"-compactor.ring.wait-stability-max-duration": "0",
+			"-compactor.ring.wait-stability-min-duration":   "0",
+			"-compactor.ring.wait-stability-max-duration":   "0",
+			"-compactor.first-level-compaction-wait-period": "0s",
```

Comment:
This is needed to unbreak `TestCompactBlocksContainingNativeHistograms`, which apparently expects this value to be lower than the new default.

Reply:
I think this is fine to not slow down the test.